applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp

```diff
@@ -197,6 +197,32 @@ class MechanicalExplicitStrategy
         return mReformDofSetAtEachStep;
     }
 
+    /**
+     * @brief This function is designed to move the mesh
+     * @note Be careful it just consider displacements, derive this method to adapt to your own strategies (ALE, FSI, etc...)
+     */
+    void MoveMesh() override
+    {
+        KRATOS_TRY
+
+        KRATOS_ERROR_IF(BaseType::GetModelPart().NodesBegin()->SolutionStepsDataHas(DISPLACEMENT_X) == false) << "It is impossible to move the mesh since the DISPLACEMENT var is not in the Model Part. Either use SetMoveMeshFlag(False) or add DISPLACEMENT to the list of variables" << std::endl;
+
+        NodesArrayType& r_nodes_array = BaseType::GetModelPart().Nodes();
+        const auto it_node_begin = r_nodes_array.begin();
+
+        #pragma omp parallel for
+        for(int i = 0; i < static_cast<int>(r_nodes_array.size()); ++i) {
+            auto it_node = it_node_begin + i;
+
+            noalias(it_node->Coordinates()) = it_node->GetInitialPosition().Coordinates();
+            noalias(it_node->Coordinates()) += it_node->FastGetSolutionStepValue(DISPLACEMENT);
+        }
+
+        KRATOS_INFO_IF("SolvingStrategy", this->GetEchoLevel() > 1 && BaseType::GetModelPart().GetCommunicator().MyPID() == 0) << " MESH MOVED " << std::endl;
+
+        KRATOS_CATCH("")
```

Comment:
why do you override this?
This looks to me like the one in the baseclass

Reply:
It avoid to print MESH MOVED. It is explicit. There are millions of steps, we know that the mesh is moved, and the buffer takes a lot of time (its super slow), and this is explicit so better to avoid it. In fact once this is merged I will propose a modification in the analysis, to not print the step and every-time. I check it, and in in small problems is increases considerably the computation time due to the buffer slowness

Reply:
I don't agree to copy-paste-code just to print some information
This is potentially a nasty bug if someone ever changes sth in the base-strategy
=> use KRATOS_WARNING as echo-level, then it will not print it (until a better solution in the Analysis is found)